src/javascript/JContent/AppLayout/ContentNavigation/NavigationHeader/LanguageSwitcher.jsx

```diff
@@ -5,15 +5,25 @@ import {useTranslation} from 'react-i18next';
 import {connect} from 'react-redux';
 import {ProgressOverlay, withNotifications} from '@jahia/react-material';
 import {useSiteInfo} from '@jahia/data-helper';
-import {cmSetAvailableLanguages} from '../JContent.redux';
-import {LanguageSwitcher} from '@jahia/design-system-kit';
 import {registry} from '@jahia/ui-extender';
+import {Dropdown} from '@jahia/moonstone';
 
-export const SiteLanguageSwitcher = ({
+function createDataObjects(languages) {
+    let data = [];
```

Comment:
Can't you use a map() for doing it in a single-line ? (and no need for a function, actually)